RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -502,6 +494,34 @@ extension OnLedgerEntity.Account.PoolUnitResources {
 	}
 }
 
+extension [OnLedgerEntity.OwnedNonFungibleResource] {
+	public var filterEmptyVaults: [OnLedgerEntity.OwnedNonFungibleResource] {
```

Comment:
I think something with nonEmptyVaults is more correct, since "filter" is ambiguous (because of how `filter` works. If we definitely want to use the word filter, the correct form would be "filtering", since this is not an in place filter method.